crates/sui-sdk/tests/tests.rs

```diff
@@ -235,3 +236,89 @@ async fn test_get_wallet_key() {
         .verify_secure(&intent_message, address, SignatureScheme::ED25519)
         .unwrap();
 }
+
+#[sim_test]
```

Comment:
I might consider using shell tests instead of simtests for this, but I think it's fine as-is